src/app.rs

```diff
@@ -442,6 +570,7 @@ impl Slot {
             let item_meta = &tile_meta.items[row][item_idx];
             ui.show_tooltip_ui("task_tooltip", &item_rect, |ui| {
                 ui.label(&item_meta.title);
+                ui.label(format!("Item UID: {}", item_meta.item_uid));
```

Comment:
Let's put this after the other fields.

Can we put this behind a toggle? It's useful debug information, but it may be confusing to end users because the UID is purely internal and has no relationship to anything the user is familiar with (or would even find in, say, the original log files).